src/hotspot/share/gc/epsilon/epsilonHeap.hpp

```diff
@@ -77,6 +77,7 @@ class EpsilonHeap : public CollectedHeap {
   virtual size_t max_capacity() const { return _virtual_space.reserved_size();  }
   virtual size_t capacity()     const { return _virtual_space.committed_size(); }
   virtual size_t used()         const { return _space->used(); }
+  virtual size_t live()         const { return used(); }
```

Comment:
I'd prefer to call `_space->used()` directly here. Minor optimization, I know.

Reply:
Right ...

Reply:
👍